templates/cart.liquid

```diff
@@ -248,10 +248,10 @@
                                 <div class="pt-4 mb-4 border-top">
                                     <h2>MY PRODUCTS</h2>
                                     <div class="border-bottom border-gray-200 py-3" ng-repeat="item in checkout.cart.configuredGroups">
-                                        <vc-checkout-configurable-line-item item="item" ></vc-checkout-configurable-line-item>
+                                        <vc-checkout-configurable-line-item item="item" on-change-qty="changeItemQty(item)" on-remove="removeItem(item)"></vc-checkout-configurable-line-item>
```

Comment:
In the review step, we shouldn't have the possibility to remove or update anything in the cart. Jira ticket has the wrong description.